include/Photo.php

```diff
@@ -324,6 +325,7 @@ public function rotate($degrees) {
 			return;
 		}
 
+		// if script dies at this point check memory_limit setting in php.ini
 		$this->image  = imagerotate($this->image,$degrees,0);
```

Comment:
I have only added the comment line as it was an important issue during testing and might help others on errors - code is untouched. I don't want to change code where I am not changing functionality. But for all other functions implemented or changed by me, I will surely follow your comments.